osu.Game/Rulesets/UI/DrawableRuleset.cs

```diff
@@ -196,13 +201,30 @@ private void load(OsuConfigManager config, CancellationToken? cancellationToken)
                 Overlays = new Container { RelativeSizeAxes = Axes.Both }
             };
 
+            var resumeInputManager = CreateInputManager();
+
             if ((ResumeOverlay = CreateResumeOverlay()) != null)
             {
-                AddInternal(CreateInputManager()
+                AddInternal(resumeInputManager
                     .WithChild(CreatePlayfieldAdjustmentContainer()
                         .WithChild(ResumeOverlay)));
             }
 
+            if (RuntimeInfo.IsMobile)
```

Comment:
why only mobile?

Reply:
oh right touch users.., not sure how I can check that?

Reply:
There's nothing to check. All users should be able to choose to use a virtual input if they want to.